src/core/middleware/security.js

```diff
@@ -21,7 +21,7 @@ export function getNoScriptStyles(
     `src/${appName}/noscript.css`,
   );
   try {
-    return fs.readFileSync(cssPath);
+    return fs.readFileSync(cssPath, 'utf8');
```

Comment:
Don't recall the error but we should really pass the encoding in `readFileSync()`